lib/gettext_i18n_rails/base_parser.rb

```diff
@@ -15,18 +15,34 @@ def self.parse(file, _options = {}, msgids = [])
       return msgids
     end
 
+    def self.libraries
+      [extension]
+    end
+
     def self.load_library
       return true if @library_loaded
 
-      begin
-        require extension
-      rescue LoadError
-        puts "A #{extension} file was found, but #{extension} library could not be found, so nothing will be parsed..."
+      loaded = libraries.find do |library|
+        begin
+          require library
+          true
+        rescue LoadError
+          false
+        end
+      end
+
+      unless loaded
+        if libraries.size == 1
+          puts "A #{extension} file was found, but #{libraries.first} library could not be found, so nothing will be parsed..."
```

Comment:
let's just go with ugly english :D 

`no library could be found #{libraries.join(', )}`